.azure-pipelines.yml

```diff
@@ -19,8 +19,4 @@ jobs:
       architecture: 'x64'
   # bin directory is in .dockerignore
   - script: |
-      python3 -m pip install --upgrade pip
-      pip3 install docker click pytest pyyaml six
-  # build a docker image and sanity test
-  - script: |
-      python3 tools/dev/dockerpkg.py build -t build --verbose --test -i cli
+    echo "Docker CI is now done in Github Actions'
```

Comment:
Nice touch here, I'm sure autopilot would take me to azure pipelines to peek at a docker build :)